ui/package.json

```diff
@@ -52,6 +52,7 @@
     "express": "^4.15.4",
     "helmet": "^3.8.1",
     "http-status": "^1.0.1",
+    "katex": "^0.8.0",
```

Comment:
Awesome -- is there a corresponding change to `npm-shrinkwrap.json`? Try running
```
docker-compose run -rm npm shrinkwrap
```
to generate it, if not.

Reply:
👍